openmp/libomptarget/plugins-nextgen/common/include/RPC.h

```diff
@@ -56,7 +55,9 @@ struct RPCServerTy {
   /// memory associated with the k
   llvm::Error deinitDevice(plugin::GenericDeviceTy &Device);
 
-  ~RPCServerTy();
+private:
+  /// Map from this device's identifier to its attached devices.
+  llvm::DenseMap<uint32_t, uintptr_t> Handles;
```

Comment:
Why a map instead of a vector?

Reply:
We could have the situation where only device 4 or something uses an RPC server, but I could probably just resize it. I'll make that change.